`:rest:spotlessMiscellaneous` is missing a dependency on `:rest:npmInstall`

If you execute as follows:

```
./gradlew assemble spotlessApply
```

you get the following failure:

```
> Task :rest:spotlessMiscellaneous FAILED

FAILURE: Build failed with an exception.

* What went wrong:
A problem was found with the configuration of task ':rest:spotlessMiscellaneous' (type 'SpotlessTaskImpl').
  - Gradle detected a problem with the following location: '/Users/davidbakin/src/hedera-mirror-node/hedera-mirror-rest/package-lock.json'.

    Reason: Task ':rest:spotlessMiscellaneous' uses this output of task ':rest:npmInstall' without declaring an explicit or implicit dependency. This can lead to incorrect results being produced, depending on what order the tasks are executed.

    Possible solutions:
      1. Declare task ':rest:npmInstall' as an input of ':rest:spotlessMiscellaneous'.
      2. Declare an explicit dependency on ':rest:npmInstall' from ':rest:spotlessMiscellaneous' using Task#dependsOn.
      3. Declare an explicit dependency on ':rest:npmInstall' from ':rest:spotlessMiscellaneous' using Task#mustRunAfter.

    Please refer to https://docs.gradle.org/8.0.2/userguide/validation_problems.html#implicit_dependency for more details about this problem.

.....

BUILD FAILED in 42s
209 actionable tasks: 29 executed, 180 up-to-date
```

(You'd like to execute the two tasks that way to save 10s+ in gradle "configuration" time.)

Also: `spotlessJavascript`, `spotlessMiscellaneous`, and `spotlessSql` - each of which take non-trivial time - execute _sequentially_.  Can they be executed in parallel? (Doesn't sound like they'd conflict with each other.)

--- a/buildSrc/src/main/kotlin/common-conventions.gradle.kts
+++ b/buildSrc/src/main/kotlin/common-conventions.gradle.kts
@@ -112,7 +112,7 @@ spotless {
         indentWithSpaces(2)
         prettier().npmExecutable(npmExecutable)
         target("**/*.json", "**/*.md", "**/*.yml", "**/*.yaml")
-        targetExclude("**/node_modules/**")
+        targetExclude("**/node_modules/**", "**/package-lock.json")
         trimTrailingWhitespace()
     })
     sql {
@@ -126,6 +126,7 @@ spotless {
         endWithNewline()
         indentWithSpaces()
         target("**/*.xml")
+        targetExclude("**/node_modules/**", "**/package-lock.json")
         trimTrailingWhitespace()
     })
 }

